draft-irtf-cfrg-aead-limits.md

```diff
@@ -413,10 +412,11 @@ integrity limits into a single expression, covered below. For this AEAD, `n =
     2^25 * (L+1) for the worst case L = |AAD|+|m| = 2; cf. Theorem 3.4.)
 -->
 ~~~
-AEA <= (v * (L + 1)) / 2^103
+CA <= 1 / 2^256
```

Comment:
Even with the assumption of an ideal PRF, an attacker should have advantage at least based on `q`.  Maybe we can just drop this line and instead say that the paper assumes that the confidentiality advantage over an ideal PRF is zero.  Of course, that is false because the PRF will repeat after a sufficient number of blocks, even if that is an implausibly large number.